src/bgen/BindingTouch.cs

```diff
@@ -83,14 +79,17 @@ public class BindingTouch : IDisposable {
 	TypeCache? typeCache;
 	public TypeCache TypeCache => typeCache!;
 
+	LibraryManager? libraryManager;
+	public LibraryManager LibraryManager => libraryManager!;
```

Comment:
Since it does not have any dependency for its constructor, lets do
```suggestion
	public LibraryManager LibraryManager { get; } = new LibraryManager ();
```
This will remove the need to use ! in other places.

Reply:
You could just make it `public LibraryManager LibraryManager => new LibraryManager();`